### PR TITLE
Removed unnecessary layers

### PR DESCRIPTION
**Pull request by florinbarbisch**

### CODE REVIEW

**Review by buesst1 — APPROVED** [review]
gud

**Comment by ghost** [comment]
# RUN: CNN, 2023-05-26 10-09-01

## Hyperparameters

| Path        | Param                                            | main                           | workspace   |
|-------------|--------------------------------------------------|--------------------------------|-------------|
| params.yaml | dvclive.data.batch_size                          | -                              | 64          |
| params.yaml | dvclive.data.num_workers                         | -                              | 0           |
| params.yaml | dvclive.data.output_size                         | -                              | 6           |
| params.yaml | dvclive.data.params.columns                      | numeric                        | -           |
| params.yaml | dvclive.data.train_val_split                     | -                              | 0.8         |
| params.yaml | dvclive.data.type                                | Sklearn_Segments               | NDArray     |
| params.yaml | dvclive.model                                    | HistGradientBoostingClassifier | CNN         |
| params.yaml | dvclive.model_hparams.learning_rate              | 0.2                            | 0.05        |
| params.yaml | dvclive.model_hparams.max_bins                   | 255                            | -           |
| params.yaml | dvclive.model_hparams.max_iter                   | 2000                           | -           |
| params.yaml | dvclive.model_hparams.max_leaf_nodes             | 31                             | -           |
| params.yaml | dvclive.model_hparams.min_samples_leaf           | 20                             | -           |
| params.yaml | dvclive.model_hparams.momentum                   | -                              | 0.9         |
| params.yaml | dvclive.model_hparams.num_epochs                 | -                              | 15          |
| params.yaml | dvclive.model_hparams.weight_decay               | -                              | 0.0005      |
| params.yaml | dvclive.seed                                     | 41                             | 40          |
| params.yaml | filter_bandpass_ord3\_low01\_high100.lowcut_f_Hz   | 1e-05                          | 0.1         |
| params.yaml | filter_highpass_ord3\_cut01.cutoff_f_Hz           | 1e-05                          | 0.1         |
| params.yaml | transform_pca.transformation_params.n_components | -                              | 6           |

## Metrics

| Path                 | Metric           | main    | workspace   | Change   |
|----------------------|------------------|---------|-------------|----------|
| dvclive/metrics.json | epoch            | -       | 15          | -        |
| dvclive/metrics.json | step             | -       | 1275        | -        |
| dvclive/metrics.json | test.epoch.acc   | 0.98289 | 0.92212     | -0.06077 |
| dvclive/metrics.json | test.epoch.f1    | 0.98085 | 0.91503     | -0.06582 |
| dvclive/metrics.json | test.step.acc    | -       | 0.92212     | -        |
| dvclive/metrics.json | test.step.f1     | -       | 0.91503     | -        |
| dvclive/metrics.json | train.epoch.acc  | 1.0     | 0.92257     | -0.07743 |
| dvclive/metrics.json | train.epoch.f1   | 1.0     | 0.9119      | -0.0881  |
| dvclive/metrics.json | train.epoch.loss | -       | 0.20035     | -        |
| dvclive/metrics.json | train.step.acc   | -       | 0.95833     | -        |
| dvclive/metrics.json | train.step.f1    | -       | 0.94376     | -        |
| dvclive/metrics.json | train.step.loss  | -       | 0.17306     | -        |
| dvclive/metrics.json | val.epoch.acc    | -       | 0.91439     | -        |
| dvclive/metrics.json | val.epoch.f1     | -       | 0.90837     | -        |
| dvclive/metrics.json | val.epoch.loss   | -       | 0.24492     | -        |
| dvclive/metrics.json | val.step.acc     | -       | 0.91439     | -        |
| dvclive/metrics.json | val.step.f1      | -       | 0.90837     | -        |
| dvclive/metrics.json | val.step.loss    | -       | 0.24492     | -        |

## Plots

### Confusion Matrix

Confusion Matrix
